packages/synapse-react-client/src/lib/style/components/_user-card-list-groups.scss

```diff
@@ -19,6 +19,7 @@
         flex-grow: 1;
         overflow-y: auto;
         button {
+          border-radius: 0px;
```

Comment:
Remove button styling, so right edge doesn't "curl up" - see "Somatic Mosaicism in the brain..." in UserCardListGroups: 
<img width="1292" alt="feature_UserCardListGroups" src="https://user-images.githubusercontent.com/26949006/231310778-b3542edb-236b-444a-b5d0-76a83f562ce6.png">